pymisp/api.py

```diff
@@ -1127,37 +1127,37 @@ def get_sharing_groups(self):
 
     def _set_user_parameters(self, **kwargs):
         user = {}
-        if kwargs.get('email'):
+        if kwargs.get('email'): 
             user['email'] = kwargs.get('email')
-        if kwargs.get('org_id'):
+        if kwargs.get('org_id') > 0:
             user['org_id'] = kwargs.get('org_id')
-        if kwargs.get('role_id'):
+        if kwargs.get('role_id') > 0:
             user['role_id'] = kwargs.get('role_id')
         if kwargs.get('password'):
             user['password'] = kwargs.get('password')
-        if kwargs.get('external_auth_required'):
+        if kwargs.get('external_auth_required') is not None:
```

Comment:
There is literally no point to these `is not None` checks.

`.get()` will return None if the key isn't found, and the value if it is.

additionally, "" is falsey. 

Hence, we can do
```python
if kwargs.get('external_auth_required'):
    user['external_auth_required'] = kwargs.get('external_auth_required')
￼```